lnwallet/errors.go

```diff
@@ -137,8 +137,8 @@ func ErrNumConfsTooLarge(numConfs, maxNumConfs uint32) error {
 // configured value for the min channel size we'll accept.
 func ErrChanTooSmall(chanSize, minChanSize btcutil.Amount) ReservationError {
 	return ReservationError{
-		fmt.Errorf("chan size of %v is below min chan size of %v",
-			chanSize, minChanSize),
+		fmt.Errorf("Your %v channel is below the %v minimum size "+
```

Comment:
Would render like:
```
Unable to open channel: Your 0.1BTC channel is below the 0.5BTC minimum size set on the connecting node
```
I think we want to start lowercase here (though other error messages are inconsistent).  How about this:
```
"channel size of %v is below the remote node's minimum size of %v"
```
Which would render like:
```
Unable to open channel: channel size of 0.1BTC is below the remote node's minimum size of 0.2BTC
```